types/font-carrier/class/font.d.ts

```diff
@@ -0,0 +1,56 @@
+import { Base } from './base';
+import { FontFace, FontFaceOptions } from './fontface';
+import { Glyph, GlyphOptions } from './glyph';
+
+export interface FontOptions {
+    /** 字体的 postscript name，默认是 1024 */
+    id: string;
+    /** 字体的水平画布大小，默认是 1024 */
+    horizAdvX: number;
+    /** 字体的垂直画布大小，默认是 1024 */
+    vertAdvY: number;
+}
+export type FontType = 'ttf' | 'eot' | 'woff' | 'woff2' | 'svg';
+
+export interface FileOptions {
+    /** 支持文件地址，也支持文件 buffer */
+    path?: string | Buffer;
+    /** 导出的字体类型 */
+    types?: FontType[];
+}
+
+/** 一个 font 对象就表示解析后的字体对象，包含字体所有的信息 */
+export class Font extends Base<FontOptions> {
```

Comment:
Does not export class Font directly but through `export = ` per https://unpkg.com/browse/font-carrier@0.3.1/lib/class/font.js

You need to have `declare class Font` and then `export = Font` instead.

Reply:
Same goes for all the other files in `front-carrier/class`

Reply:
thanks, Please check it again